### PR TITLE
[OIS] Add to flake8 in workflow and fix python files (part #25193) 

### PR DESCRIPTION
**Pull request by DamMicSzm**
This pull request is a part of #25193  ( part fix python files in Open IoT SDK )

### Problem

Python files need prevent things like syntax errors, typos, bad style, etc... it saves time for reviewing your code. Many python files needed bug fixes.

### Changes

Fix all python files in Open IoT SDK platform where linter find problem. Adding Open IoT SDK platform to check with flake8.

### Testing

CI will test and maybe some another manual testing

### CODE REVIEW

File: .github/workflows/lint.yml [hunk 1]
```diff
@@ -57,7 +57,6 @@ jobs:
               run: scripts/checkout_submodules.py --shallow --platform linux
             - name: Bootstrap cache
               uses: actions/cache@v3
-              timeout-minutes: 10
```
Comment (bzbarsky-apple) [inline .github/workflows/lint.yml:60]: Why is this being removed?

**Review by bzbarsky-apple — COMMENTED** [review]
Please get the openiotsdk folks to review this.

File: .github/workflows/lint.yml (hunk 1 above)
Comment (DamMicSzm) [reply .github/workflows/lint.yml]: Sorry it is my mistake. Thank you

**Comment by DamMicSzm** [comment]
@ATmobica please review

**Review by ATmobica — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
@pan- can you review please?

**Review by pan- — APPROVED** [review]
All good for me @bzbarsky-apple

**Review by bzbarsky-apple — APPROVED** [review]

**Comment by bzbarsky-apple** [comment]
Fast-tracking platform-specific change with platform owner review.

**Pull request merged**